{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/graphql/users/mutations.py

```diff
@@ -4,9 +4,10 @@
 from graphene import relay
 from graphql import GraphQLError
 
-from .types import CurrentUser, AuthenticatedUser
 from {{cookiecutter.main_module}}.users import services as user_services
-from {{cookiecutter.main_module}}.users.auth import tokens, services as auth_services
+from {{cookiecutter.main_module}}.users.auth import tokens
+from {{cookiecutter.main_module}}.users.auth import services as auth_services
+from .types import AuthenticatedUser, CurrentUser
```

Comment:
I'd suggest being consistent with imports like:

```suggestion
from .{{cookiecutter.main_module}}.users.types import AuthenticatedUser, CurrentUser
```

Reply:
This .types file is not in users app, but at the same level as the current (mutations.py) file.